app/views/users/edit.html.erb

```diff
@@ -0,0 +1,13 @@
+<%if @user.errors.any?%>
+  <p>The fields are mandotory</p>
+  <% @user.errors.full_messages.each do |e|%>
+    <ul style="color:red"><%= e %></ul>
+    <% end %>
+<% end%>
+
+<%= form_for @user do |f| %>
+  Username:</br><%= f.text_field :username, placeholder: "Your username" %></br>
+  Email:</br> <%= f.text_field :email, placeholder: "example@example.com" %></br>
+  Password:</br> <%= f.text_field :password, placeholder: "Your password" %></br>
```

Comment:
The type of input you are using for password (text_field) is not the right one. There is one for passwords to avoid the input from showing the text you type and protect it.